internal/service/advancedcluster/resource_advanced_cluster_test.go

```diff
@@ -555,12 +560,12 @@ func TestAccClusterAdvancedClusterConfig_symmetricShardedOldSchema(t *testing.T)
 		CheckDestroy:             acc.CheckDestroyCluster,
 		Steps: []resource.TestStep{
 			{
-				Config: configShardedOldSchemaMultiCloud(orgID, projectName, clusterName, 2, "M10"),
-				Check:  checkShardedOldSchemaMultiCloud(clusterName, 2, "M10", false),
+				Config: configShardedOldSchemaMultiCloud(orgID, projectName, clusterName, 2, "M10", &configServerManagementModeFixedToDedicated),
```

Comment:
1. Choosing to only test for the old schema as there is extra logic there.
2. Choosing to re-use existing test as we need a sharded cluster (also picked a test without migration tests since we are introducing a new field)